exercises/practice/rna-transcription/RnaTranscriptionTest.php

```diff
@@ -1,58 +1,66 @@
 <?php
 
-/*
- * By adding type hints and enabling strict type checking, code can become
- * easier to read, self-documenting and reduce the number of potential bugs.
- * By default, type declarations are non-strict, which means they will attempt
- * to change the original type to match the type specified by the
- * type-declaration.
- *
- * In other words, if you pass a string to a function requiring a float,
- * it will attempt to convert the string value to a float.
- *
- * To enable strict mode, a single declare directive must be placed at the top
- * of the file.
- * This means that the strictness of typing is configured on a per-file basis.
- * This directive not only affects the type declarations of parameters, but also
- * a function's return type.
- *
- * For more info review the Concept on strict type checking in the PHP track
- * <link>.
- *
- * To disable strict typing, comment out the directive below.
+/**
+ * @uuid 123e4567-e89b-12d3-a456-426614174000
  */
-
-declare(strict_types=1);
-
 class RnaTranscriptionTest extends PHPUnit\Framework\TestCase
 {
     public static function setUpBeforeClass(): void
     {
         require_once 'RnaTranscription.php';
     }
 
+    /**
+     * @testdox It transcribes guanine to cytosine
+     * @uuid 123e4567-e89b-12d3-a456-426614174001
```

Comment:
In all the doc blocks above the tests:

`@uuid` is no valid annotation. We use a comment `uuid` only. Please change that.

```suggestion
     * uuid 123e4567-e89b-12d3-a456-426614174001
```